src/components/AppHelp.vue

```diff
@@ -0,0 +1,255 @@
+				</div>
+				<div class="feature icon-fullscreen">
+					{{ t('notes', 'Write down your thoughts without any distractions.') }}
+				</div>
+				<div class="feature icon-files-dark">
+					{{ t('notes', 'Organize your notes in categories.') }}
+				</div>
+				<button class="button-icon-add icon-add" @click="onNewNote">
+					{{ t('notes', 'Create a sample note with markdown') }}
+				</button>
+			</AppSettingsSection>
+			<AppSettingsSection :title="t('notes', 'Markdown')">
+				<div class="feature icon-toggle-filelist">
+					{{ t('notes', 'Use Markdown markups to style your text.') }}
+				</div>
+				<button @click="onNewNote">
+					{{ t('notes', 'Create a new Note with markdown samples') }}
+				</button>
+				<table>
+					<tr>
+						<th>
+							{{ t('notes', 'Sequence') }}
+						</th>
+						<th>
+							{{ t('notes', 'Result') }}
+						</th>
+						<th>
+							{{ t('notes', 'Visualized') }}
+						</th>
+					</tr>
+					<tr v-for="item in getMarkdown">
+						<td v-html="item.sequence">
+							{{ item.sequence }}
+						</td>
+						<td>
+							{{ item.result }}
+						</td>
+						<td v-html="item.visualized">
+							{{ item.visualized }}
+						</td>
+					</tr>
+				</table>
+			</AppSettingsSection>
+			<AppSettingsSection :title="t('notes', 'Shortcuts')">
+				<div class="feature icon-toggle-filelist">
+					{{ t('notes', 'Use shortcuts to quickly navigate this app.') }}
+				</div>
+				<table>
+					<tr>
+						<th>{{ t('notes', 'Shortcut') }}</th>
+						<th>{{ t('notes', 'Action') }}</th>
+					</tr>
+					<tr v-for="item in getShortcuts">
+						<td>{{ item.shortcut }}</td>
+						<td>{{ item.action }}</td>
+					</tr>
+				</table>
+			</AppSettingsSection>
+			<AppSettingsSection :title="t('notes', 'Tips and Tricks')">
+				<div class="feature icon-toggle-filelist">
+					{{ t('notes', 'Double Click the text in viewmode to quickly open edit mode!') }}
+				</div>
+			</AppSettingsSection>
+			<AppSettingsSection :title="t('notes', 'Apps')">
+				<div class="feature icon-phone">
+					{{ t('notes', 'Install the app for your mobile phone in order to access your notes from everywhere.') }}
+				</div>
+				<div class="badge-wrapper">
+					<a href="https://github.com/stefan-niedermann/nextcloud-notes">
+						{{ t('notes', 'Android app: Nextcloud Notes by Niedermann IT-Dienstleistungen ') }}
+					</a>
+					<div>
+						<div class="badge">
+							<a target="_blank" href="https://play.google.com/store/apps/details?id=it.niedermann.owncloud.notes">
+								<img :src="getRoute('badge_playstore.svg')" class="appstore-badge badge-playstore-fix">
+							</a>
+						</div>
+						<div class="badge">
+							<a target="_blank" href="https://f-droid.org/repository/browse/?fdid=it.niedermann.owncloud.notes">
+								<img :src="getRoute('badge_fdroid.svg')" class="appstore-badge">
+							</a>
+						</div>
+					</div>
+				</div>
+				<div class="badge-wrapper">
+					<a href="https://github.com/phedlund/CloudNotes">
```

Comment:
I think we should open all external links in a new window/tab.
```suggestion
					<a target="_blank" href="https://github.com/phedlund/CloudNotes">
```